src/Components/Header/Header.tsx

```diff
@@ -1,19 +1,21 @@
 import React, { ChangeEvent, useState } from 'react';
 import PropTypes from 'prop-types';
-import { Logo } from '../Logo';
+import { Link, useLocation } from 'react-router-dom';
 import { Movie } from '../MovieCard';
 import { ModalWithForm } from '../ModalWithForm';
 import { CongratulationsModal } from '../CongratulationsModal';
 import { ModalWindowWrapper } from '../ModalWindowWrapper';
 import { useToggle } from '../../Utilities';
 import css from './Header.less';
 
-export function Header({ addMovie, searchMovie }:
-  { addMovie: (newMovie: Movie) => void, searchMovie: (searchString: string) => void },
-) {
+export function Header({ addMovie }: { addMovie: (newMovie: Movie) => void }) {
+  const { search } = useLocation();
+  const queryParam = search.match(new RegExp('[?&]searchString=([^&]+).*$'));
```

Comment:
The task is a little bit easier because URL has the following format `search/:searchQuery` instead of query params

Consider using `useQuery` instead of `useLocation` to receive query parameters if you decided to use queries

Reply:
fixed